src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -248,12 +257,28 @@ export const WorkspaceDashboard = _.flow(
           ]),
           ..._.map(({ membersGroupName }) => div({ style: styles.authDomain }, [membersGroupName]), authorizationDomain)
         ]),
+        div({ style: styles.header }, ['Notifications']),
+        h(Fragment,
+          _.map(({ notificationKey, description }) => h(LabeledCheckbox, {
```

Comment:
Since there's only one of these right now, the `map` seems unnecessary. If we do add another item later, we'll need to touch the code here anyway, so might as well keep things simple for now.

Reply:
@zarsky-broad

Reply:
See above...